broadcast_formats.md

```diff
@@ -81,20 +88,20 @@ This data format will be removed in 2.x branch firmware.
 Format 4 was the primary data format in RuuviTags shipped before June 2018. It is described in detail [here](./dataformat_04.md).
 
 # Data format 5 (also known as RAWv2)
-**Lifecycle: Beta testing.**
+**Lifecycle: In production.**
```

Comment:
We'll mark this as in production once the factory actually ships the tags with the data format 5

Reply:
Oops, I thought they were shipping.

Reply:
Add something like:

Has been available for download and flash  since 2.2.2, May 26, 2018 at  https://github.com/ruuvi/ruuvitag_fw/releases

Reply:
Opps. Availability note is already included.